ui/app/AppLayouts/Wallet/popups/swap/SwapSignApprovePopup.qml

```diff
@@ -351,7 +351,6 @@ StatusDialog {
                 Layout.maximumWidth: 60
                 titleText: qsTr("Max slippage:")
                 infoText: "%1%".arg(LocaleUtils.numberToLocaleString(root.swapSignApproveInputForm.selectedSlippage))
-                loading: root.loading
```

Comment:
No longer needed?

Reply:
it was a mistake max slippage should not reload during refresh!
https://www.figma.com/design/TS0eQX9dAZXqZtELiwKIoK/Swap---Milestone-1?node-id=4151-71714&m=dev